src/ontology/Makefile

```diff
@@ -39,15 +41,18 @@ SUBSETDIR=                  subsets
 SCRIPTSDIR=                 ../scripts
 UPDATEREPODIR=              target
 SPARQLDIR =                 ../sparql
+EXTENDED_PREFIX_MAP=        $(TMPDIR)/obo.epm.json
 COMPONENTSDIR =             components
 ROBOT_PROFILE =             profile.txt
 REPORT_FAIL_ON =            None
 REPORT_LABEL =              
 REPORT_PROFILE_OPTS =       --profile $(ROBOT_PROFILE)
-OBO_FORMAT_OPTIONS =        
+OBO_FORMAT_OPTIONS =        --clean-obo "strict drop-untranslatable-axioms"
 SPARQL_VALIDATION_CHECKS =  properties-as-annotation-and-object ordo-mapping-annotations 
 SPARQL_EXPORTS =            basic-report 
-ODK_VERSION_MAKEFILE =      v1.5.4
+ODK_VERSION_MAKEFILE =      v1.6
+RELAX_OPTIONS =             --include-subclass-of true
```

Comment:
Are there QC tests/review that should be done for the goals `$(ONT)-base.owl` and `$(ONT)-full.owl` with this addition?

Reply:
I don't think we need this in this repo.. I can't think of any good reason at least..

Reply:
If there was a big effect it would be on the subclass sync. We already ran a build - we know there is not one. So imo that's good enough testing..